cmd/cluster-controller/main.go

```diff
@@ -15,6 +15,7 @@ var (
 	kubeconfigPath = flag.String("kubeconfig", "", "Path to kubeconfig")
 	syncerImage    = flag.String("syncer_image", "", "Syncer image to install on clusters")
 	pullModel      = flag.Bool("pull_model", true, "Deploy the syncer in registered physical clusters in POD, and have it sync resources from KCP")
+	pushModel      = flag.Bool("push_model", false, "If true, run syncer for each cluster from inside cluster controller")
```

Comment:
Maybe `pull_mode` and `push_mode` would make more sense. I feel like the initial `pull_model` arg name was quite the wrong word when I introduced it.
WDYT ?

Reply:
Yeah good call. I was thinking about doing it in a followup but it was easy enough to do in here.